src/main/kotlin/uk/gov/gdx/datashare/services/EventDataService.kt

```diff
@@ -133,9 +133,11 @@ class EventDataService(
     return EventNotification(
       eventId = event.id,
       eventType = subscription.eventType,
-      sourceId = event.dataId,
+      eventTime = event.eventTime,
+      ingestTime = event.whenCreated,
+      sourceId = if (includeData && EnrichmentField.SOURCE_ID in enrichmentFieldNames) event.dataId else null,
       dataIncluded = if (!callbackEvent) includeData else null,
-      enrichmentFields = if (!callbackEvent) enrichmentFieldNames else null,
+      enrichmentFields = enrichmentFieldNames,
```

Comment:
Think we should always return these

Reply:
I'd quite like not to return these at all for `/events/{id}` where they're present, but can't figure out how

Reply:
yes I notice this - it shows null which is annoying